sdk/webpubsub/azure-messaging-webpubsubservice/azure/messaging/webpubsubservice/_patch.py

```diff
@@ -266,7 +278,7 @@ def _configure(
         self.authentication_policy = kwargs.get('authentication_policy')
         if self.credential and not self.authentication_policy:
             if isinstance(self.credential, AzureKeyCredential):
-                self.authentication_policy = JwtCredentialPolicy(self.credential, kwargs.get('user'))
+                self.authentication_policy = JwtCredentialPolicy(self.credential, **kwargs)
```

Comment:
I would pull out the specific parameters rather than passing through all `kwargs`. We are instantiating an internal type that we know about, so no pass-through of arbitrary stuff is needed. If we were to to anything with `**kwargs` in the `JwtCredentialPolicy`, passing through makes more sense. Not blocking the PR.